purchase_allowed_product/README.rst

```diff
@@ -74,7 +74,10 @@ Contributors
 
   * Darius Žižys
 
-* Ooops404 <info@ooops404.com>
+* `Ooops404 <https://www.ooops404.com>`__:
+  * Ooops404 <info@ooops404.com>
```

Comment:
This is not a contributor per se, so if you put the company name, don't put this line.

Reply:
@pedrobaeza https://github.com/OCA/maintainer-tools/issues/592

@GSLabIt I think if you remove line 78 it should be good to go

Reply:
Exactly